testsuite/features/profiles/cloud_aws/Kiwi/POS_Image-JeOS7_head/root/etc/systemd/system/image-deployed.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Notify Uyuni about newly deployed image
```

Comment:
Not Uyuni but SUSE Manager, when applies.

Reply:
This would need to be changed in the template. This is just a copy.

Reply:
Can't we change manually this text after generating the profiles from the template?
It's a one time change.

Reply:
You can write a "sed" command to the Makefile. Otherwise it would be overwritten when the next one call "make"

Reply:
ahh, no. It is easier. We just change the template